src/evaluation/WorkspaceManager.jl

```diff
@@ -15,7 +15,7 @@ Each notebook gets at most one `Workspace` at any time, but it can also have no
 (it cannot `eval` code in this case).
 """
 Base.@kwdef mutable struct Workspace
-    pid::Integer
+    pid::Int
```

Comment:
Could you check that `Distributed.addprocs` indeed returns an `Int`, not `Int64` on a 32 bit system?

Reply:
Yes it is `Int` on 1.8~x86:
```
julia> using Distributed; typeof(Distributed.addprocs(1))
Vector{Int32} (alias for Array{Int32, 1})

```